manpage/wf-recorder.1.scd

```diff
@@ -74,6 +80,44 @@ wf-recorder - A simple screen recording program for wlroots-based compositors
 *wf-recorder* is a tool built to record your screen on Wayland compositors.
 It makes use of wlr-screencopy for capturing video and FFmpeg for encoding it.
 
+To select a specific part of the screen you can either use `-g <geometry>`, or
+use [slurp](https://github.com/emersion/slurp) for interactive selection of the
```

Comment:
```suggestion
use https://github.com/emersion/slurp for interactive selection of the
```

This is because man does not how links like github does.

Reply:
Thanks. I meant to unlink this. I read `man scdoc` and noticed it doesn't seem to support Markdown hyperlinks.